atmsimulator/login.py

```diff
@@ -46,7 +46,9 @@ def login():
             customer_category_selection()
             clear()
 
-            res = requests.put(f"http://localhost:3000/users/{user_name}", 
+            user_id = data[0].get("id")

```

Comment:
I mentioned that the json server is run with `--id username` flag and argument, this eliminates the need for your `data[0].get("id")` doesn't it.